backend/canisters/user_index/impl/src/queries/users.rs

```diff
@@ -63,7 +63,8 @@ fn users_impl(args: Args, state: &RuntimeState) -> Response {
                 .map(|u| UserSummaryV2 {
                     user_id: u.user_id,
                     stable: (u.date_updated > updated_since).then(|| u.to_summary_stable(now)),
-                    volatile: (u.date_updated_volatile > updated_since).then(|| u.to_summary_volatile(now)),
+                    volatile: (updated_since == 0 || u.date_updated_volatile > updated_since)
```

Comment:
I think 0 shouldn't be a special case. Passing in a timestamp before any users were created should be the same as passing in 0.

Reply:
Why won't it work for newly created users?

Reply:
User1 is created at T1. User2 requests users updated since T1 - 1. Volatile data for user1 isn't returned.

Reply:
I guess it's fine if the default volatile data is what would've been returned

Reply:
> User1 is created at T1. User2 requests users updated since T1 - 1. Volatile data for user1 isn't returned.

The FE requests users since 0 if that user isn't in the cache